src/board_controller/brainflow_boards.cpp

```diff
@@ -710,14 +710,15 @@ BrainFlowBoards::BrainFlowBoards()
     {
         {"name", "BrainAlive"},
         {"sampling_rate", 250},
-        {"timestamp_channel", 15},
-        {"marker_channel", 16},
+        {"timestamp_channel", 16},
+        {"marker_channel", 17},
         {"package_num_channel", 0},
-        {"num_rows", 17},
+        {"num_rows", 18},
         {"eeg_channels", {1, 2, 3, 4, 5, 6, 7, 8}},
         {"eeg_names", "F7,FT7,T7,CP5,CZ,C6,FC6,F4"},
         {"accel_channels", {9, 10, 11}},
-        {"ppg_channels", {12, 13, 14}}
+        {"gyro_channels", {12, 13, 14}},
+        {"packet_id", {15}}
```

Comment:
we use package_num_channel for packet id, I see no reason to duplicate them. Even more we do not have an API to query for `packet_id`, its only for `package_num_channel` and its unified among boards. Some device specific data can be written to `other_info` channels, but here its clearly should be `package_num_channel`